eZ/Publish/API/Repository/Values/Content/TrashItem.php

```diff
@@ -13,4 +13,10 @@
  */
 abstract class TrashItem extends Location
 {
+    /**
+     * Trashed timestamp.
+     *
+     * @var mixed Trashed timestamp.
+     */
+    protected $trashed;
```

Comment:
I'm rather against timestamps in favor of `DateTime`. 

On the other hand, the one in `SPI\Persistence` should remain as a timestamp and be should be mapped to proper `DateTime`.